frame/header-mmr/src/lib.rs

```diff
@@ -172,19 +179,18 @@ impl<T: Trait> MMRStore<T::Hash> for ModuleMMRStore<T> {
 impl<T: Trait> Module<T> {
 	impl_rpc! {
 		pub fn gen_proof_rpc(
-			block_number: T::BlockNumber,
-			mmr_block_number: T::BlockNumber,
+			block_number_of_member_leaf: u64,
+			block_number_of_last_leaf: u64,
 		) -> RuntimeDispatchInfo<T::Hash> {
```

Comment:
What if using short names of `block_number_of_member_leaf` and `block_number_of_last_leaf`, and complete the concepts in docs?

```rust
/// * member: block_number_of_member_leaf
/// * last_leaf: block_number_of_last_leaf
pub fn gen_proof_rpc(member: u64, last_leaf: u64) {}
```

The long arguments are easy to understand but hard to read.